models/replyModel.js

```diff
@@ -0,0 +1,40 @@
+const { Schema, model } = require("mongoose");
+
+const replySchema = Schema(
+  {
+    id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    user_id: {
+      type: String,
+      required: true,
+    },
+    discussion_id: {
+      type: Schema.Types.ObjectId,
+      required: false,
+      ref: "Discussion",
+    },
+    reply_id: {
+      type: String,
+      required: false,
+    },
+    reply: {
```

Comment:
this is not supposed to be here

Reply:
yes, I just figured while following the uml diagram that there's no actual place to add text to replies
we need to redo this meeting, I don't really understand that part sha
but should I remove it?

Reply:
I'll remove it